### PR TITLE
[Durable Objects] Fixing broken links

### PR DESCRIPTION
**Pull request by Maddy-Cloudflare**
PCX-7389

### CODE REVIEW

File: content/workers/configuration/bindings/_index.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ KV namespace bindings allow for communication between a Worker and a KV namespac
 
 Durable Object bindings for communication between a Worker and a Durable Object.
 
-* Learn more about [Durable Object bindings](/workers/runtime-apis/durable-objects/#access-a-durable-object-from-a-worker).
-* Configure Durable Object bindings via your [`wrangler.toml` file](/workers/wrangler/configuration/#durable-objects).
+* Learn more about [Durable Object bindings](/durable-objects/how-to/access-durable-object-from-a-worker/).
+* Configure Durable Object bindings via your [`wrangler.toml` file](/durable-objects/get-started/#5-configure-durable-object-bindings).
```
Comment (deadlypants1973) [inline content/workers/configuration/bindings/_index.md]:
```suggestion
* Configure Durable Object bindings via your [`wrangler.toml` file](/workers/wrangler/configuration/#durable-objects).
```
Until there is a more comprehensive page on bindings within the DO docs, I think the Wrangler docs have more effective instructions. Or update the DO get started page to include the wrangler config link for Durable Objects (/workers/wrangler/configuration/#durable-objects) somewhere.

Comment (deadlypants1973) [reply content/workers/configuration/bindings/_index.md:38]: The codeblock here also has a duplicate that needs to be fixed: https://developers.cloudflare.com/durable-objects/get-started/#5-configure-durable-object-bindings

File: content/workers/wrangler/workers-kv.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Note that binding names do not need to correspond to the namespace you created.
...
+A binding is a how your Worker interacts with external resources such as [KV Namespaces](/workers/runtime-apis/kv/), [Durable Objects](/durable-objects/), or [R2 Buckets](/r2/api/workers/workers-api-reference/). A binding is a runtime variable that the Workers runtime provides to your code. You can declare a variable name in your `wrangler.toml` file that will be bound to these resources at runtime, and interact with them through this variable. Every binding's variable name and behavior is determined by you when deploying the Worker. Refer to the [Environment Variables](/workers/configuration/environment-variables/) documentation for more information.
```
Comment (deadlypants1973) [inline content/workers/wrangler/workers-kv.md:50]:
```suggestion
A binding is a how your Worker interacts with external resources such as [KV Namespaces](/workers/runtime-apis/kv/), [Durable Objects](/durable-objects/), or [R2 Buckets](/r2/api/workers/workers-api-reference/). A binding is a runtime variable that the Workers runtime provides to your code. You can declare a variable name in your `wrangler.toml` file that will be bound to these resources at runtime, and interact with them through this variable. Every binding's variable name and behavior is determined by you when deploying the Worker. Refer to the [Environment Variables](/workers/configuration/environment-variables/) documentation for more information.
```

**Review by deadlypants1973 — COMMENTED** [review]
PCX

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**